src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.2.json

```diff
@@ -128,13 +128,13 @@
                 {
                     "QuestionId": "YO-22",
                     "QuestionTag": null,
-                    "Label": "<span class='govuk-label'>Company name</span>",
+                    "Label": "Company name",
                     "ShortLabel": "",
                     "QuestionBodyText": "",
                     "Hint": "",
                     "Input": {
                         "Type": "Text",
-                        "InputClasses": "app-uppercase",
+                        "InputClasses": "govuk-!-width-two-thirds",
```

Comment:
app-uppercase missing?

Reply:
We don't want to force them to put their company name in block capitals do we?

Reply:
Yes, that was the requirement on the original story. It's returned in upper case by the UKRLP and Companies House services

Reply:
Okay. I've put it back in 👌